productivity-suite/app/fragments/news/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "productivity-suite-news-fragment",
+  "scripts": {
+    "dev": "solid-start dev",
+    "build": "solid-start build",
+    "start": "npx wrangler dev --local"
+  },
+  "type": "module",
+  "main": "./dist/index.js",
+  "devDependencies": {
+    "solid-start-cloudflare-workers": "^0.2.0",
+    "solid-start-node": "^0.2.0",
+    "typescript": "^4.8.4",
+    "vite": "^3.1.8",
+    "wrangler": "^2.1.13",
+		"shared": "file:../../shared"
```

Comment:
this is misaligned 🙈

Reply:
it appears that this package.json is using 2-space and the one I copy pasted from uses tabs :(

Reply:
Let's add the prettifier to this branch and demo too...

Reply:
If it's ok I'll do that in a follow-up PR to avoid merge conflicts with Dario's message bug PR